student.rb

```diff
@@ -1,9 +1,17 @@
 require_relative 'person'
 
 class Student < Person
+  attr_reader :classroom
+
   def initialize(classroom, age, name = 'Unknown', parent_permissions: true)
     super(age, name, parent_permissions: parent_permissions)
     @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
```

Comment:
- [ ] Kindly remove this line of code in your initialize method. Your setter method `classroom=(classroom)` will handle this. 👍

Reply:
Sure, one question related to this.
@Gambit142  Does the setter get call when we first initialize the class?

Reply:
- [ ] @wandergithub, you don't have to initialize the classroom for this project. Besides, you have a method to set the classroom for a student in the `classroom=(classroom)` method.